community-operators/bundles/metering/bundle.0.13.0.yaml

```diff
@@ -0,0 +1,551 @@
+                          failureThreshold: 3
+                          httpGet:
+                            path: /readiness
+                            port: 44135
+                            scheme: HTTP
+                          initialDelaySeconds: 1
+                          periodSeconds: 10
+                          successThreshold: 1
+                          timeoutSeconds: 1
+                        resources:
+                          limits:
+                            cpu: 50m
+                            memory: 100Mi
+                          requests:
+                            cpu: 50m
+                            memory: 50Mi
+                      restartPolicy: Always
+                      securityContext:
+                        runAsNonRoot: true
+                      serviceAccount: metering-operator
+                      terminationGracePeriodSeconds: 30
+        customresourcedefinitions:
+          owned:
+          - description: An instance of Metering
+            displayName: Metering
+            kind: Metering
+            name: meterings.metering.openshift.io
+            version: v1alpha1
+          - description: A table within PrestoDB
+            displayName: Metering Presto Table
+            kind: PrestoTable
+            name: prestotables.metering.openshift.io
+            version: v1alpha1
+          - description: A resource describing a source of data for usage by Report Generation
+              Queries
+            displayName: Metering data source
+            kind: ReportDataSource
+            name: reportdatasources.metering.openshift.io
+            version: v1alpha1
+          - description: A SQL query used by Metering to generate reports
+            displayName: Metering generation query
+            kind: ReportGenerationQuery
+            name: reportgenerationqueries.metering.openshift.io
+            version: v1alpha1
+          - description: A Prometheus query by Metering to do metering
+            displayName: Metering prometheus query
+            kind: ReportPrometheusQuery
+            name: reportprometheusqueries.metering.openshift.io
+            version: v1alpha1
+          - description: A metering report summarizes metrics based on the query specified
+            displayName: Metering Report
+            kind: Report
+            name: reports.metering.openshift.io
+            version: v1alpha1
+          - description: Represents a configurable storage location for Metering to store metering
+              and report data
+            displayName: Metering storage location
+            kind: StorageLocation
+            name: storagelocations.metering.openshift.io
+            version: v1alpha1
+      
+  packages: |-
+    - #! package-manifest: deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml
+      packageName: metering
+      channels:
+      - name: alpha
+        currentCSV: metering-operator.v0.12.0
```

Comment:
Just realized, this should be pointing to the new CSV. The metering-operator's `package.yaml` should be updated as well.

Reply:
> Just realized, this should be pointing to the new CSV. The metering-operator's package.yaml should be updated as well.